test/blackbox-tests/test-cases/github11527.t/dune

```diff
@@ -0,0 +1,3 @@
+(executable
+ (name bug)
+ (public_name bug))
```

Comment:
Does it need a `public_name`? To me it seems `(executable (name bug))` or so should be enough?

Reply:
It does not, I'll simplify things. This is leftover from the initial issue.